src/render_tasks/d3d12_path_tracer.hpp

```diff
@@ -218,16 +217,10 @@ namespace wr
 			}
 
 			// Wait for AS to be built
-			{
-				auto barrier = CD3DX12_RESOURCE_BARRIER::UAV(as_build_data.out_tlas.m_native);
-				cmd_list->m_native->ResourceBarrier(1, &barrier);
-			}
+			d3d12::UAVBarrierAS(cmd_list, as_build_data.out_tlas, frame_idx);

```

Comment:
double UAV Barrier? (line 203 already has a barrier, maybe my misunderstanding?)

Reply:
your right, Just modified the existing code and didn't change anything. fixing now